subsys/bluetooth/controller/hci_driver.c

```diff
@@ -521,6 +521,12 @@ static int configure_supported_features(void)
 		}
 	}
 
+	if (IS_ENABLED(CONFIG_BT_CTLR_CX_ADV_TRY_CONTINUE_ON_DENIAL)) {
```

Comment:
Please also add `if (IS_ENABLED(CONFIG_BT_BROADCASTER))` to minimize the flash footprint.

Reply:
Would it be cleaner to add that in KConfig?

Reply:
I'm fine with either